tests/e2e/utils/lambda_layer/powertools_layer.py

```diff
@@ -31,6 +38,11 @@ def build(self) -> str:
 
         return str(self.output_dir)
 
+    def after_build(self):
+        subprocess.run(self.cleanup_command, shell=True)
+
+        super(LocalLambdaPowertoolsLayer, self).after_build()
```

Comment:
I don't think we need that? Please correct me if I'm wrong.

We're implementing `BaseLocalLambdaLayer` ABC, and specifically `after_build`.

If a future class subclass `LocalLambdaPowertoolsLayer` and wants to extend `after_build`, then this would be valid.

Reply:
we don't, I was just trying to be a good citizen :) Deleting it